src/AzureExtension/Services/DevBox/PackagesService.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using AzureExtension.Contracts;
+using Microsoft.Windows.DevHome.SDK;
+using Log = AzureExtension.DevBox.Log;
+
+namespace AzureExtension.Services.DevBox;
+
+public class PackagesService
+{
+    private Windows.Management.Deployment.PackageManager _packageManager = new();
+
+    public bool IsPackageInstalled(string packageName)
+    {
+        var currentPackage = _packageManager.FindPackagesForUser(string.Empty, packageName).FirstOrDefault();
+        return currentPackage != null;
```

Comment:
Now that I'm thinking about it, if the Windows App is a package thats in the store, aren't there already App Model class available that we can use to check if the package is installed on the computer without using WinGet?

Reply:
What's wrong with the current implementation?

Reply:
Oh it was mostly about whether it was the right approach for this simple scenario. I was thinking we didn't need to use Winget since there are already built in Windows APIs to check if a package was installed. For example using the windows deployment manager https://learn.microsoft.com/en-us/uwp/api/windows.management.deployment.packagemanager.findpackage?view=winrt-22621.

But if you need to one day have the ability to install a package for a user then we might as well use WinGet.

Reply:
But It'd also be preferrable to make this class implement an interface if we're adding it to the host container. We want to minimize the concrete classes in the Host Container. But I still think we probably don't need this as a class. If we can use the api above, then it can just be a method call in the Dev Box instance instead.

Reply:
I'm still a little confused. The API used is the third one below from your link. 
And this class is an interface (last commit) to mock.

Reply:
Thanks for clarifying offline. Have pushed the latest changes.